src/test/java/bio/terra/service/snapshotbuilder/utils/SearchConceptsQueryBuilderTest.java

```diff
@@ -21,16 +21,22 @@
 class SearchConceptsQueryBuilderTest {
 
   private SnapshotBuilderDomainOption createTestSnapshotBuilderDomainOption(
```

Comment:
This name seems redundant to me. I would omit the terms `test` and `snapshotBuilder` since they are implied by the package and class name.
```suggestion
  private SnapshotBuilderDomainOption createDomainOption(
```